packages/liveblocks-client/rollup.config.js

```diff
@@ -186,7 +186,10 @@ export default async () => {
   const external = [
     ...Object.keys(pkgJson?.dependencies ?? {}),
     ...Object.keys(pkgJson?.peerDependencies ?? {}),
-  ];
+  ].flatMap((dep) =>
+    // Also include `@liveblocks/.../internal` as an external reference, by convention
+    dep.startsWith("@liveblocks/") ? [dep, dep + "/internal"] : [dep]
+  );
```

Comment:
After restoring this `"exports"` section, I noticed the CommonJS build started suffering from an issue, where `@liveblocks/client/internal` would no longer be recognized as an "external" reference any longer. This somehow wasn't a problem for the ESM build.

Therefore, I have added this "fix" in our Rollup builds, to consider both `@liveblocks/client` _and_ `@liveblocks/client/internal` as external refs.

This had the following effect on the build output:
https://gist.github.com/nvie/938f21ae8bb8c5daca1287553a816605